public/app/features/admin/UserOrgs.tsx

```diff
@@ -255,3 +275,68 @@ export class AddToOrgModal extends PureComponent<AddToOrgModalProps, AddToOrgMod
     );
   }
 }
+
+interface UnThemedChangeOrgButtonProps extends Themeable {
+  isExternalUser?: boolean;
+  onChangeRoleClick: () => void;
+  onCancelClick: () => void;
+  onOrgRoleSave: () => void;
+}
+
+interface UnThemedChangeOrgButtonState {
+  isChangingRole: boolean;
+}
+
+const getChangeOrgButtonTheme = (theme: GrafanaTheme) => ({
+  disabledTooltip: css`
+    display: flex;
+  `,
+  tooltipItemLink: css`
+    color: ${theme.palette.blue95};
+  `,
+});
+
+export class UnThemedChangeOrgButton extends PureComponent<UnThemedChangeOrgButtonProps, UnThemedChangeOrgButtonState> {
+  render() {
+    const { isExternalUser, theme } = this.props;
+    const styles = getChangeOrgButtonTheme(theme);
+
+    return (
+      <div className={styles.disabledTooltip}>
+        <ConfirmButton
+          confirmText="Save"
+          onClick={this.props.onChangeRoleClick}
+          onCancel={this.props.onCancelClick}
+          onConfirm={this.props.onOrgRoleSave}
+          disabled={isExternalUser}
+        >
+          Change role
+        </ConfirmButton>
+        {isExternalUser && (
+          <Tooltip
+            placement="right-end"
+            content={
+              <div>
+                This user&apos;s role is not editable because it is synchronized from your auth provider. Refer to
+                the&nbsp;
+                <a
+                  className={styles.tooltipItemLink}
+                  href={'https://grafana.com/docs/grafana/latest/auth'}
+                  rel="noreferrer"
+                  target="_blank"
+                >
+                  Grafana authentication docs
+                </a>
+                &nbsp;for details.
+              </div>
+            }
+          >
+            <Icon name="question-circle" />
+          </Tooltip>
+        )}
+      </div>
+    );
+  }
+}
+
+const ChangeOrgButton = withTheme(UnThemedChangeOrgButton);
```

Comment:
```ts
interface ChangeOrgButtonProps {
  isExternalUser?: boolean;
  onChangeRoleClick: () => void;
  onCancelClick: () => void;
  onOrgRoleSave: () => void;
}

export function ChangeOrgButton({
  isExternalUser,
  onChangeRoleClick,
  onOrgRoleSave,
  onCancelClick,
}: ChangeOrgButtonProps): ReactElement {
  const styles = useStyles2(getChangeOrgButtonTheme);
  return (
    <div className={styles.disabledTooltip}>
      <ConfirmButton
        confirmText="Save"
        onClick={onChangeRoleClick}
        onCancel={onCancelClick}
        onConfirm={onOrgRoleSave}
        disabled={isExternalUser}
      >
        Change role
      </ConfirmButton>
      {isExternalUser && (
        <Tooltip
          placement="right-end"
          content={
            <div>
              This user&apos;s role is not editable because it is synchronized from your auth provider. Refer to
              the&nbsp;
              <a
                className={styles.tooltipItemLink}
                href={'https://grafana.com/docs/grafana/latest/auth'}
                rel="noreferrer"
                target="_blank"
              >
                Grafana authentication docs
              </a>
              &nbsp;for details.
            </div>
          }
        >
          <Icon name="question-circle" />
        </Tooltip>
      )}
    </div>
  );
}

const getChangeOrgButtonTheme = (theme: GrafanaTheme2) => ({
  disabledTooltip: css`
    display: flex;
  `,
  tooltipItemLink: css`
    color: ${theme.v1.palette.blue95};
  `,
});
```